pyfolio/ipycompat.py

```diff
@@ -0,0 +1,19 @@
+import IPython
```

Comment:
Talked about this in person.  If we add a `utils` directory, I could see moving this there.  I don't think this belongs in single `utils` file.